src/AppBundle/Facade/FaqFacade.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace AppBundle\Facade;
+
+use AppBundle\Entity\Faq;
+use AppBundle\Repository\FaqRepository;
+
+class FaqFacade
+{
+
+    private $faqRepository;
+
+    public function __construct(FaqRepository $faqRepository)
+    {
+	$this->faqRepository = $faqRepository;
+    }
+
+    public function getAll()
+    {
+	return $this->faqRepository->findBy(
+			[], [
+		    "question" => "asc"
```

Comment:
Tady je nějak rozházený indent... nevím jestli bych řadil zrovna podle odpovědi, ale nijak jsme to neřešili, takže ok